api/controllers/export-data.js

```diff
@@ -121,7 +121,13 @@ module.exports = {
     //    by the following auth check in ctx.district (maybe?)
     //  - Still don't let offline users use this API, and instead refactor the
     //    export logic so it can be used in webapp, and have exports works offline
-    return auth.check(req, ['national_admin', getExportPermission(req.params.type)])
+    return auth.getUserCtx(req)
+      .then(userCtx => {
+        if (!auth.isOnlineOnly(userCtx)) {
+          throw new Error({ code: 403, message: 'Insufficient privileges' });
```

Comment:
Unfortunately, this will end up looking like this in console:
![image](https://user-images.githubusercontent.com/35681649/41350430-cde9fd6c-6f1b-11e8-8b2d-9c5668cf3238.png)

Reply:
You're absolutely right! Good spotting.